flycheck-pos-tip.el

```diff
@@ -82,16 +103,34 @@ Uses `pos-tip-show' under the hood."
   "Hide the Flycheck tooltip."
   (pos-tip-hide))
 
+(defvar flycheck-pos-tip-tty-old-message nil
+  "The contents of the message buffer before showing errors.")
+
+(defun flycheck-pos-tip-tty-show (messages)
+  "Show MESSAGES in the echo area."
+  (setq flycheck-pos-tip-tty-old-message (current-message))
+  (message "%s" (mapconcat #'identity messages "\n\n")))
+
+(defun flycheck-pos-tip-tty-hide ()
+  "Restore old message in echo area."
+  (when flycheck-pos-tip-tty-old-message
+    (message flycheck-pos-tip-tty-old-message)))
+
 ;;;###autoload
 (defun flycheck-pos-tip-error-messages (errors)
-  "Display ERRORS in a graphical tooltip."
+  "Display ERRORS in a graphical tooltip, if possible."
```

Comment:
"if possible" -> "on a GUI frame".

I'd also add something along the following: "On TTY frames it falls back to …".